site/mkdocs.yml

```diff
@@ -94,6 +94,8 @@ nav:
   - Format:
     - Definitions: terms.md
     - Spec: spec.md
+  - Roadmap:
+    - Overview: roadmap-overview.md
```

Comment:
I think we can put this in the first section "Project". Any reason for a separated section?

Reply:
I was initially planning to have more sub-sections, but ended up having only a single one. Will move it under `Project`